test/azure/Expected/AcceptanceTests/Lro/lro/aio/operations_async/_lro_retrys_operations_async.py

```diff
@@ -120,11 +121,13 @@ async def put201_creating_succeeded200(
             'polling_interval',
             self._config.polling_interval
         )
-        raw_result = await self._put201_creating_succeeded200_initial(
-            product=product,
-            cls=lambda x,y,z: x,
-            **kwargs
-        )
+        cont_token = kwargs.pop('continuation_token', None)  # type: Optional[str]
+        if cont_token is None:
```

Comment:
So in the "resume" scenario, nothing will be awaited in this function - right?

Reply:
correct